File: backend/simulator.py
import random


def get_sensor_data():
    return {
        "air_temperature": round(random.uniform(24, 30), 1),
        "humidity": random.randint(70, 90),
        "soil_temperature": round(random.uniform(22, 27), 1),
        "soil_moisture": random.randint(50, 75),
        "water_level": random.randint(65, 90),
        "air_quality": "GOOD"
    }

File: backend/simulator.js
function getSensorData() {
    return {
        airTemperature: Number((Math.random() * 6 + 24).toFixed(1)),
        humidity: Math.floor(Math.random() * 21 + 70),
        soilTemperature: Number((Math.random() * 5 + 22).toFixed(1)),
        soilMoisture: Math.floor(Math.random() * 26 + 50),
        waterLevel: Math.floor(Math.random() * 26 + 65),
        airQuality: "GOOD"
    };
}

module.exports = getSensorData;